### PR TITLE
AccountSwitcher: Support for importing accounts in bulk

### PR DESCRIPTION
**Pull request by archubbuck**

### CODE REVIEW

**Comment by xKylee** [comment]
This doesn’t look like it adds the accounts via the encryption method, in which case it will need to be reworked to do so

**Comment by xKylee** [comment]
I would leave the existing panel as is, and have this on config option with a new button appearing if set to true to import cases.

**Comment by xKylee** [comment]
Can you also confirm what format this requires for importing?

**Comment by archubbuck** [comment]
@xKylee Sure, no problem. Just to make sure I'm understanding you correctly, you would prefer that there be a toggle added to the configuration options that shows/hides the new button above the label field?

The import format is "{label}:{username}:{password}".

**Comment by xKylee** [comment]
> @xKylee Sure, no problem. Just to make sure I'm understanding you correctly, you would prefer that there be a toggle added to the configuration options that shows/hides the new button above the label field?
> 
> The import format is "{label}:{username}:{password}".

Correct. I would also add a warning onto the config option advising the format of which you will need to import it. Also, what happens if your password/username/label utilises “:”?

**Comment by xKylee** [comment]
@archubbuck you ok making them changes?

Have you also tested adding multiple accounts in 1 hit with this method?

**Comment by archubbuck** [comment]
Hi @xKylee. I'm happy to make those changes. Work was crazy today but I plan to have these changes completed before the weekend.

File: runelite-client/src/main/java/net/runelite/client/plugins/profiles/ProfilesPanel.java [hunk 1]
```diff
@@ -243,7 +268,6 @@ public void focusLost(FocusEvent e)
 
 		txtPasswordLogin.setEchoChar((char) 0);
 		txtPasswordLogin.setForeground(ColorScheme.LIGHT_GRAY_COLOR);
-		txtPasswordLogin.setToolTipText(PASSWORD_LABEL);
```
Comment (ThatGamerBlue) [inline runelite-client/src/main/java/net/runelite/client/plugins/profiles/ProfilesPanel.java]: why was this removed?

**Comment by zeruth** [comment]
This is a very niche feature, that seems kind of redundant... Enter logins into a text file -> load into plugin. When you could just skip that step and enter logins directly to the plugin.

Then there is the moral side of it, where this ends up being a QOL feature for people attempting to hijack accounts from leaked databases etc.

So for those two reasons, we are going to deny this PR. Feel free to host this in your own repo when our external rework goes live soon™️
